opentibiabr/otservbr-global
# Restructuring of the quest storages

**# Description**

Restructuring of the quest storages IDs and how to organize it, allowing bigger ranges between each update to accommodate new quests to be written and revision of old style quests.

With the this restructuration, old servers that has already keys storages saved to the database will pass thru a script to migrate old IDs into the new IDs.

As example of this PR, the quests from Pre update 6 to update 7.1 was already migrated.

The tables to update as long new quests are revised is in:
data/startup/tables/storage_keys_update.lua

**## How to test**
Start any server with the regular main datapack, execute some quests in your map that is present in any of the versions listed above.
Apply the PR to the server and restart, while the server start the old keys stored to the database should then be migrated to the new IDs.


## Checklist
